pkg/service/wire_gen.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by Wire. DO NOT EDIT.
 
-//go:generate go run -mod=mod github.com/google/wire/cmd/wire
+//go:generate go run github.com/google/wire/cmd/wire
```

Comment:
Guessing my install and somebody else's wire install are clashing. Let me install deps again.